povertymapping/feature_engineering.py

```diff
@@ -93,6 +95,7 @@ def generate_features(
 
     # Get list of features generated
     feature_cols = [x for x in aoi.columns if x not in input_cols]
+    feature_cols = feature_cols+["pop_count"] if use_hdx else feature_cols
```

Comment:
might have to update this later for adding population density

nit: might be easier to read with just an if statement, no need for the else :)

Reply:
a cleaner implementation is to simply get all features (population not being optional anymore) but agree with this implementation for backwards compatibility!

Reply:
Got this. Adding `POP_VARIABLES` in `hdx.py` and using an `if` statement instead.

Reply:
Changes in this thread are reverted as previously I used that workaround since the `pop_count` column was both a feature and part of the `input_cols`. With the new `population_density` column, this workaround is not needed anymore.